public/javascript/posts/BookAppointment.js

```diff
@@ -19,6 +19,24 @@ var app = angular.module('BookAppointmentApp');
     ];
 
     $scope.viewRenderWeekAgenda = function(view, element){

```

Comment:
you have got the complete element here (which is html ement actually) , please use the `fc-body` from the elemet to add a tooltip